src/Microsoft.Azure.Relay/RelayedHttpListenerContext.cs

```diff
@@ -129,7 +129,8 @@ internal async Task RejectAsync(Uri rendezvousUri)
 
         IClientWebSocket CreateWebSocket()
         {
-            var clientWebSocket = ClientWebSocketFactory.Create(this.Listener.UseBuiltInClientWebSocket);
+            var clientWebSocket = ClientWebSocketFactory.Create(this.Listener.UseBuiltInClientWebSocket,
```

Comment:
>Create [](start = 57, length = 6)

Style: All parameters on a single line or all on their own line:

Use
ClientWebSocketFactory.Create(\r\n
    this.Listener.UseBuiltInClientWebSocket, this.Listener.CustomClientWebSocketFactory);

or 

ClientWebSocketFactory.Create(\r\n
    this.listener.UseBuiltInClientWebSocket,\r\n
    this.listener.CustomClientWebSocketFactory);